server/src/models/NoteModel.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  Default,
+  Description,
+  Email,
+  Enum,
+  Example,
+  Format,
+  Ignore,
+  Maximum,
+  MaxLength,
+  Minimum,
+  MinLength,
+  Pattern,
+  Required,
+  Title,
+} from '@tsed/common'
+import { Indexed, Model, ObjectID, Unique } from '@tsed/mongoose'
+
+// TODO: Edit model in accordance to the DB data
+
+enum Categories {
+  CAT1 = 'cat1',
+  CAT2 = 'cat2',
+}
+
+@Model()
+export class NoteModel {
```

Comment:
Is it a WIP model?

Reply:
It is. I'll update it accordingly with DB data when structured.